modules/tinymce/src/themes/silver/test/ts/headless/components/tree/TreeTest.ts

```diff
@@ -138,6 +138,99 @@ describe('headless.tinymce.themes.silver.tree.TreeTest', () => {
     await UiFinder.pWaitFor('Wait for menu item to show up', SugarBody.body(), '[title="menuitem"]');
     Mouse.clickOn(SugarBody.body(), '[title="menuitem"]');
     store.assertEq('menuitem', [ 'menuitem' ]);
+
+    Mouse.clickOn(dir.element, '.tox-tree--directory .tox-trbtn.tox-tree--directory__label');
+    assertDirectoryExpandedState('Collapsed', false, getTreeItem('.tox-tree--directory .tox-tree--directory'));
+
   });
 
+  it('TINY-9640: Tree is navigable with keyboard according to a11y recommendations from mdn', () => {
```

Comment:
There is detailed explanation in the form of comments in the test already and it would be too long to mention how arrow keys work so I just mentioned w3c which is more official than mdn